### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by remi-dupre**
Replaces Travis with Github actions.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        with:
+          python-version: '3.8'
+
+      - name: Display Python version
+        run: python -c "import sys; print(sys.version)"
+
+      - name: Install dependancies
+        run: |
+          pip install pipenv
+          pipenv install --dev
```
Comment (amatissart) [inline .github/workflows/tests.yml:23]: I would keep  the `--deploy` option to validate and use the `Pipfile.lock` file to install dependencies
```suggestion
          pipenv install --dev --deploy
```

Comment (remi-dupre) [reply .github/workflows/tests.yml:23]: That's probably the only think that I could have forgotten ... --'

Comment (amatissart) [reply .github/workflows/tests.yml:23]: you force-pushed, but no change is visible :slightly_smiling_face:

Comment (remi-dupre) [reply .github/workflows/tests.yml]: Well, that's interesting ^^'

**Review by amatissart — APPROVED** [review]

**Pull request merged**